Gruntfile.js

```diff
@@ -0,0 +1,145 @@
+module.exports = function(grunt) {
+    grunt.initConfig({
+        concurrent: {
+            serverAndTest: ["nodemon", "jasmine-node"],
+            options: {
+                logConcurrentOutput: true
+            }
+        },
+        nodemon: {
+            dev: {
+                // NEVER PUT A PATH IN THIS FILE NAME
+                script: "server.js",
+                options: {
+                    // PUT PATH HERE INSTEAD
+                    // THIS TOOK ME 4 HOURS TO FIX
+                    cwd: __dirname + "/server"
+                }
+            }
+        },
+        karma: {
+            continuous: {
```

Comment:
I should be able to change this to "dev:" since it's just the name of a target, and that would make it consistent with the nodemon target.